### PR TITLE
Adds table populated with data from local file.

### PR DESCRIPTION
**Pull request by helrond**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Pull requests accepted!
 
 Hillel Arnold  
 Julia Welby  
-Emeline Swanson
+Emeline Swanson  
 Erich Chang
```
Comment (EcZX) [inline README.md]: Erich Chang

Comment (EcZX) [inline README.md]: TESTING

Comment (EcZX) [inline README.md]: This is terrible.

Comment (jwelbs) [inline README.md]: I'm cool with the order.

**Pull request merged**